### PR TITLE
refactor(dev): alpine.js => hono/jsx/dom

### PR DESCRIPTION
**Pull request by tmm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,9 +37,13 @@
...
     "tsx": "^4.7.1",
-    "typed-htmx": "^0.2.1",
     "typescript": "^5.3.3",
     "vitest": "^1.2.2"
   },
-  "packageManager": "pnpm@8.15.3"
+  "packageManager": "pnpm@8.15.3",
+  "pnpm": {
+    "overrides": {
+      "hono": "4.0.9"
```
Comment (tmm) [inline package.json]: forcing all packages to use same hono version during dev

File: src/frog-base.tsx [hunk 2]
```diff
@@ -380,19 +380,7 @@ export class FrogBase<
...
-        // check if devtools are enabled on constructor.
-        (this.dev?.enabled ?? true) &&
-        // check if route has `/dev` path.
-        this.hono.routes.some((r) => {
-          const currentFullPath =
-            (this.basePath === '/' ? '' : this.basePath) + parsePath(path)
-          if (!r.path.startsWith(currentFullPath)) return false
-          if (!r.path.includes('/dev')) return false
-          return true
-        })
```
Comment (jxom) [inline src/frog-base.tsx:410]: Do we still need this for edge environments? Think `happy-dom` and `@farcaster/core` were the constraining packages in the devtools.

Comment (tmm) [reply src/frog-base.tsx:410]: going to drop those

**Comment by tmm** [comment]
closing for https://github.com/wevm/frog/pull/156

**Pull request closed without merging**